frontera/contrib/scrapy/converters.py

```diff
@@ -7,6 +7,8 @@
 from frontera.utils.converters import BaseRequestConverter, BaseResponseConverter
 from w3lib.util import to_bytes, to_native_str
 
+from frontera.utils.misc import dict_to_unicode, dict_to_bytes
+
 
 class RequestConverter(BaseRequestConverter):
```

Comment:
Please have a look at https://github.com/scrapinghub/frontera/issues/211#issuecomment-251931413
I am not very sure if this is the best place to resolve the issue. Although fixing this at `from_frontier` would solve the issue, my concern is that an encoder-decoder should not change the original form of a message on decoding.